app/views/posts/index.html.erb

```diff
@@ -1 +1,23 @@
-<h1>This will show up information about all posts for a given user</h1>
\ No newline at end of file
+
+<div class="posts-section">
+ <%= render partial: 'users/user_card', locals: {user: @user} %>
+ <ul class="post-list">
+    <% @user.posts.each_with_index do |post, i| %>
+     <li >
+       <ul>
+         <li>
+         <%= render partial: 'posts/post_card', locals: {post: post, post_number: i+1} %>
+         </li>
+         <li>
+             <%= render partial: 'application/comments_box', locals: { post: post  }%>
+         </li>
+        </ul>
+     </li>
+    <% end %>
```

Comment:
- [ ] Similar to what I pointed out above, in this post index page, you are not meant to display all the comments for each post. You are meant to use the method you created earlier `five_recent_comments` and display only 5 comments on each post. I suggest you update this correctly. :+1: :100: